db.go

```diff
@@ -288,11 +296,75 @@ func (connector *DbConnector) GetChecksCount() (int64, error) {
 	return ts, err
 }
 
+// GetUsernameID read ID of user by login
+func (connector *DbConnector) GetUsernameID(login string) (string, error) {
+	if len(login) > 0 && login[0] == byte('#') {
+		result := "@" + login[1:]
+		log.Debugf("Channel %s requested. Returning id: %s", login, result)
+		return result, nil
+	}
+
+	c := connector.Pool.Get()
+	defer c.Close()
+
+	result, err := redis.String(c.Do("GET", fmt.Sprintf("moira-users:%s", login)))
```

Comment:
Same namespace problem here. Imagine that somebody wants to add similar functionality to other sender, like Slack. User ids will be different, but you already took the prefix `moira-users` for Twitter users. Not fair :(